server/startup/migrations/v109.js

```diff
@@ -1,16 +1,20 @@
 RocketChat.Migrations.add({
```

Comment:
I think this is not needed anymore.. it is now on version 113

Reply:
Reversed file deletion.